dockerfiles/base/scripts/base/startup_funcs.sh

```diff
@@ -403,12 +527,17 @@ start() {
   set -- "${@/\-\-debug/}"
   set -- "${@/\-\-offline/}"
   set -- "${@/\-\-trace/}"
+  set -- "${@/\-\-skip\:preflight/}"
+  set -- "${@/\-\-skip\:postflight/}"
+  set -- "${@/\-\-skip\:nightly/}"
+  set -- "${@/\-\-skip\:network/}"
+  set -- "${@/\-\-skip\:pull/}"
   
-    # The pre_init method is unique to each assembly. This method must be provided by 
+  # The cli_post_init method is unique to each assembly. This method must be provided by 
   # a custom CLI assembly in their container and can set global variables which are 
   # specific to that implementation of the CLI. This method must be called after
   # networking has been established and initial images downloaded.
-  cli_pre_init
+  cli_post_init
```

Comment:
Yeah, that is the expected flow for me.  We have two types of init - `init` and `cli_init`.  And in this situation the `pre` and `post` reflect the actions before and after the bootstrap phase, which is the `init` but not the `cli_init`.

I think what we call `cli_init` is just another phase of the init process.  So instead of it being just a single lump sum activity, we break the initialization into five phases.  Maybe we should rename all of the phases to be explicit names instead of pre / post.